nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -205,25 +205,38 @@ def __init__(
         else:
             self.out_proj = None
             self._feat_out = d_model
+        self.xmax = pos_emb_max_len
+        device = next(self.parameters()).device
+        seq_range = torch.arange(0, self.xmax, device=device)
+        self.register_buffer('seq_range', seq_range, persistent=False)
 
     @typecheck()
     def forward(self, audio_signal, length=None):
+        audio_size: int = audio_signal.size(-1)
+        if audio_size > self.xmax:
+            logging.warning(
+                f"Audio Signal is too long : {audio_size}, please increase pos_emb_max_len (={pos_emb_max_len}) config parameter!"
```

Comment:
They can not change that parameter from the config easily. We need to add a method to update this parameter for them.

Reply:
Agreed. Look into how this supported for ConvASREncoder inside https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/asr/parts/mixins/mixins.py#L153
And
https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/asr/parts/utils/asr_module_utils.py

Reply:
I have added set method in this PR.